README_EN.md

```diff
@@ -0,0 +1,45 @@
+# Tebukuro
+[![wercker status](https://app.wercker.com/status/35e08da3b0a742e5c50ffac2670a821b/s/master "wercker status")](https://app.wercker.com/project/byKey/35e08da3b0a742e5c50ffac2670a821b)
+
+ "Mobile Friendly" open-source web service for managing events and communities.
+
+<img src="http://4.bp.blogspot.com/-dMHKByNfZxo/Uku9_iuHWHI/AAAAAAAAYqg/LjlcTLrN44o/s800/winter_tebukuro.png" width="320px">
+
+## What is tebukuro?
+
+Tebukuro is a "Mobile Friendly" open-source web service, enabling you to manage your communities and events solely from mobile devices. Furthermore, not only is there support for announcements and reception before events, but for communication during events and ones that have already closed as well.
+
+## Features
+
+- **Mobile Friendly**
+  - [ ] Manage communities and events from mobile devices
+
+- **Energize your communities**
+  - [ ] Simple procedure for releasing new events to communities' site
```

Comment:
I see, so it's the community's page.

I suggest writing this:
```
Simple procedure for opening new events in communities.
```

I think this is a simple way of saying what you want to say.